src/subscriptions/SubscriptionManager.cpp

```diff
@@ -174,8 +174,8 @@ SubscriptionManager::subLedger(
     boost::asio::yield_context& yield,
     std::shared_ptr<WsBase> session)
 {
-    subscribeHelper(session, ledgerSubscribers_, [this](session_ptr session) {
-        unsubLedger(session);
+    subscribeHelper(session, ledgerSubscribers_, [this](session_ptr sesh) {
```

Comment:
if session at 175 not used , can we do 
`session_ptr _)`

Reply:
See prior comment above regarding removing session and assorted consequences. Regarding session_ptr sesh, it is a necessary parameter for subscription and unsubscription methods within SubscriptionManager.cpp